api/src/main/java/io/opentelemetry/tags/DefaultTagger.java

```diff
@@ -149,4 +150,22 @@ public TagMap fromByteArray(byte[] bytes) {
       return EmptyTagMap.INSTANCE;
     }
   }
+
+  private static final class TagMapInScope implements Scope {
```

Comment:
Suggestion: Consider `ScopedTagMap` to be consistent with `ScopedSpan`.

Reply:
https://github.com/open-telemetry/opentelemetry-java/blob/master/api/src/main/java/io/opentelemetry/trace/DefaultTracer.java#L182